cnf-certification-test/performance/suite.go

```diff
@@ -49,34 +49,34 @@ var _ = ginkgo.Describe(common.PerformanceTestKey, func() {
 
 	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestRtAppNoExecProbes)
 	ginkgo.It(testID, ginkgo.Label(tags...), func() {
-		var guaranteedPodContainersWithExclusiveCPUs = env.GetGuaranteedPodContainersWithExlusiveCPUs()
+		var guaranteedPodContainersWithExclusiveCPUs = env.GetGuaranteedPodContainersWithExclusiveCPUs()
 		testhelper.SkipIfEmptyAll(ginkgo.Skip, guaranteedPodContainersWithExclusiveCPUs)
 		testRtAppsNoExecProbes(&env, guaranteedPodContainersWithExclusiveCPUs)
 	})
 
 	// Scheduling related tests begins here
 	testID, tags = identifiers.GetGinkgoTestIDAndLabels(identifiers.TestSharedCPUPoolSchedulingPolicy)
 	ginkgo.It(testID, ginkgo.Label(tags...), func() {
-		env.Pods = env.GetPodsWithoutHostPID() // Consider only pods with no access to host machine
-		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Pods)
+		// Consider only pods with no access to host machine
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.GetPodsWithoutHostPID())
```

Comment:
Since the test skips if any are empty, this should work
testhelper.SkipIfEmptyAny(ginkgo.Skip, env.GetPodsWithoutHostPID(), env.GetNonGuaranteedPodContainers())